### ISSUE TITLE
BERT demo --fp32 Unable to achieve accelerated optimization results in reports

### ISSUE THREAD

**Issue by ye1024** [NVIDIA/TensorRT#742]
Description

While repeating the demo of BERT base
python3 builder.py -m /workspace/TensorRT/demo/BERT/models/fine-tuned/bert_tf_v2_base_fp16_128_v2/model.ckpt-8144 -o /workspace/TensorRT/demo/BERT/bert.engine -b 1 -s 384
and than run 
python3 infer.py -e ./bert.engine -b 1 -v /home/chenjiasheng/zhixuda/bert_base/vocab.txt -p "TensorRT is a high performance deep learning inference platform that delivers low latency and high throughput for apps such as recommenders, speech and image/video on NVIDIA GPUs. It includes parsers to import models, and plugins to support novel ops and layers before applying optimizations for inference. Today NVIDIA is open-sourcing parsers and plugins in TensorRT so that the deep learning community can customize and extend these components to take advantage of powerful TensorRT optimizations for your apps." -q "What is TensorRT?" -s 384 -i 100 -w 50

I get follow output:
Question: What is TensorRT?
Running 100 iterations with Batch Size: 1
        Total Time: 1157.3128957748413 ms       Average Time: 11.573128957748413 ms     95th Percentile Time: 11.643936157226562 ms     99th Percentile Time: 11.648032188415527

I want to know why the output is so different from the published results
Thanks！

Environment
TensorRT Version: 7.1.3.4 (tar installation)
GPU Type:Tesla P100 PCIE 16GB
Nvidia Driver Version:  450.51.06
CUDA Version: 11.0
CUDNN Version: 8.0
Operating System + Version: ubuntu 16.04 
Python Version (if applicable): 3.6.0
TensorFlow Version (if applicable): 1.14

**Comment by ttyio** [comment]
Hi @ye1024 , the `inference.py` is mainly used for functional test. We measure perf using `perf.py`, and we can build a binary `build/perf` if you want to try cuda graph.

The perf data in README is collected from a more friendly command wrapper `bash scripts/inference_benchmark.sh --gpu <arch>` . Could you take a try? thanks.

**Comment by ttyio** [comment]
Hello @ye1024 , we will close since get no response for more than 3 weeks, please reopen if you still see perf mismatch, thanks
